src/electron/emain.ts

```diff
@@ -534,43 +534,44 @@ electron.ipcMain.on("restart-server", (event) => {
         runWaveSrv();
     }
     event.returnValue = true;
-    return;
 });
 
 electron.ipcMain.on("reload-window", (event) => {
     if (MainWindow != null) {
         MainWindow.reload();
     }
     event.returnValue = true;
-    return;
 });
 
-electron.ipcMain.on("open-external-link", async (_, url) => {
-    try {
-        await electron.shell.openExternal(url);
-    } catch (err) {
-        console.warn("error opening external link", err);
-    }
+electron.ipcMain.on("open-external-link", (_, url) => {
```

Comment:
Not sure, my linter was complaining about it, I think there's a concern about leaking promises but I don't remember exactly. I'll do a bit more research

Reply:
@sawka it seems like it's mostly a semantic concern, though it can result in unhandled errors if we're not careful. I think the right move is to create a fireAndForget wrapper that handles the promise rejection.